### DISCUSSION TITLE
non riesco a firmare

### DISCUSSION THREAD

**Discussion by danyx67800** [iisgiua/giuaschool#769] [Richieste di aiuto]
quando vado nella sezione registro non c'è l'opzione per firmare

**Comment by danyx67800** [comment]
@trinko e di conseguenza non posso aggiungere voti

**Comment by trinko** [comment]
Succede a qualsiasi utente? Per qualsiasi ora? 
In tal caso è un problema dovuto alla configurazione: controlla nei parametri le date di inizio/fine anno e di fine periodi intermedi. 
Controlla anche le date degli scrutini in **SCUOLA > SCRUTINI**.
